README.md

```diff
@@ -27,16 +27,34 @@ docker run --rm -it ksprojects/zkcopy --source server:port/path --target server:
 ## Options
 
 ```
- -c,--copyOnly <true|false>               (optional) set this flag if you
-                                          do not want to remove nodes that
-                                          are removed on source
- -h,--help                                print this message
- -i,--ignoreEphemeralNodes <true|false>   (optional) set this flag if you
-                                          do not want to copy ephemeral
-                                          ZNodes
- -s,--source <server:port/path>           location of a source tree to
-                                          copy
- -t,--target <server:port/path>           target location
- -w,--workers <N>                         (optional) number of concurrent
-                                          workers to copy data
-```
\ No newline at end of file
+Usage: zkcopy [-ci] [--help] [--timeout=<sessionTimeout>] [-b=<batchSize>]
+              [-m=<mtime>] -s=server:port/path -t=server:port/path
+              [-w=<workers>]
+      --help                  display this help and exit
+      --timeout=<sessionTimeout>
+                              Session timeout in milliseconds
+                                Default: 40000
+  -b, --batchSize=<batchSize> Batch write operations into transactions of this
+                                many operations. Batch sizes are limited by the
+                                jute.maxbuffer server-side config, usually
+                                around 1 MB.
+                                Default: 1000
+  -c, --copyOnly[=<copyOnly>] set this flag if you do not want to remove nodes
+                                that are removed on source
+  -i, --ignoreEphemeralNodes[=<ignoreEphemeralNodes>]
+                              set this flag to false if you do not want to copy
+                                ephemeral ZNodes
+  -m, --mtime=<mtime>         Ignore nodes older than mtime
+                                Default: -1
+  -s, --source=server:port/path
+                              location of a source tree to copy
+                                Default: 10-81-26-99-uswest2cdevc:
```

Comment:
As far as I see there's no default value in the code (and there should not be one), please update readme.

Reply:
Good spot, I think that must have snuck in when we generated the help with some auto-defaults in there.